modules/ding_provider/connie_search/includes/connie_search.search.inc

```diff
@@ -196,3 +196,28 @@ function connie_search_search_sort_options() {
     ]
   ];
 }
+
+/**
+ * Provide standard object ID's for connie provider.
+ *
+ * @param mixed $ids
+ *    Array with multiple ID's or a single ID.
+ *
+ * @return array|string
+ *    Array with multiple ID's. ID/faust as key and object ID as value.
+ *    If a string is provided as parameter, a string with object ID.
+ */
+function connie_search_search_object_ids($ids) {
```

Comment:
You need to add a test-case to DingSearchProviderImplementationTestCase over in ding_provider_implementation.test
- Add a new abstract getTranslatedObjectIds() function in DingSearchProviderImplementationTestCase
- Implement the function in ConnieSearchSearchProviderImplementationTestCase (have it return the expected list of id's)
- Implement a testObjectIds() function in DingSearchProviderImplementationTestCase and have it invoke provider_invoke('search', 'object_ids') with a list of id's and test that connie returns a translated list that is equal to invoking $this->getTranslatedObjectIds()